ls1intum/Artemis
# Improve the breadcrumb navigation for the exam exercise tutor dashboard

### Checklist

### Motivation and Context

Currently going to the exercise tutor dashboard brings you from the exam to the course, breaking breadcrumb navigation.

### Description

Moves the routes to have the dashboard as child route of the exam. Also fixes a crash when opening the Assessment Dashboard when there are no exercise groups.

### Steps for Testing

1. Navigate to an exam and visit the Assessment Dashboard
2. Open an Exercise Dashboard from there
3. Make sure no errors occur and you can still use the dashboard as usual
4. Open the Assessment Dashboard in an exam without exercise groups and check that there are no errors
5. Open Tutor and Instructor Assessment Dashboards of a course and make sure they work as on develop

### Screenshots

The breadcrumbs now fully display the exam route instead of a course route:
![grafik](https://user-images.githubusercontent.com/72132281/110215789-e74ca780-7eab-11eb-821c-f5567709f3ef.png)
